service/worker/scanner/workflow.go

```diff
@@ -79,7 +79,8 @@ var (
 		TaskList:                     historyScannerTaskListName,
 		ExecutionStartToCloseTimeout: infiniteDuration,
 		WorkflowIDReusePolicy:        cclient.WorkflowIDReusePolicyAllowDuplicate,
-		CronSchedule:                 "0 */12 * * *",
+		CronSchedule:                 "*/15 * * * *",
```

Comment:
How about making it a dynamic config?

Reply:
Whoops, I wasn't planning on including that change in the patch. OTOH I don't see much advantage in letting these accumulate for 12 hours.

It wouldn't do much good as a dynamic config, because the workflow schedule isn't actually changed unless you explicitly terminate that workflow and let it be recreated.

Let me just put this back to original value for this for this bug fix. Making it changeable should probably be separate work.

Reply:
yeah that sounds good too.